.github/actions/kots-e2e/action.yml

```diff
@@ -160,6 +160,10 @@ runs:
         kubectl -n $VELERO_NAMESPACE logs daemonset/node-agent
         echo "------previous node-agent logs"
         kubectl -n $VELERO_NAMESPACE logs -p daemonset/node-agent
+        echo "------velero backup details"
+        curl -LO https://github.com/vmware-tanzu/velero/releases/download/v1.12.1/velero-v1.12.1-linux-amd64.tar.gz
```

Comment:
should this download latest instead of a specific version?

Reply:
removed